PURL-SPECIFICATION.rst

```diff
@@ -273,7 +279,7 @@ How to build ``purl`` string from its components
 Building a ``purl`` ASCII string works from left to right, from ``type`` to
 ``subpath``.
 
-Note: some extra type-specific normalizations are required.
+Note: some extra type-specific normalizations are REQUIRED.
```

Comment:
This is an impossible requirement if the addition of new types is considered to be not a breaking change. More practically, normalization is REQUIRED during canonicalization and comparison, but otherwise implementations should do their best.